rdrf/rdrf/templates/rdrf_cdes/alert_language_is_not_supported.html

```diff
@@ -0,0 +1,34 @@
+{% load i18n %}
+{% load get_language_settings_codes %}
+{% get_language_settings_codes as LANGUAGES %}
+
+<script type="text/javascript">
+    $(document).ready(function(){
+        const browser_language = navigator.language;
+        const supported_languages = {{ LANGUAGES|safe }};
+        const is_browser_language_in_settings = [browser_language, browser_language.split('-')[0]].some((language) => supported_languages.includes(language));
+        const existing_not_support_language = localStorage.getItem('not_support_language');
+        const dismiss_alert = `dismiss_language_${browser_language}_not_support_alert`;
+        const get_dismiss_alert_value = localStorage.getItem(dismiss_alert);
+
+        if (!is_browser_language_in_settings && get_dismiss_alert_value === null) {
+            const language_name_converter = new Intl.DisplayNames(['en'], { type: 'language' });
+            $('#id_language_not_support_alert').text(gettext(`Language "${language_name_converter.of(browser_language)}" is not currently supported.`))
```

Comment:
Can we make this alert text its own template / block so that it's easy to override in registry repositories.